static/app/components/group/replayCount.tsx

```diff
@@ -0,0 +1,59 @@
+import styled from '@emotion/styled';
+
+import Placeholder from 'sentry/components/placeholder';
+import {IconPlay} from 'sentry/icons';
+import DiscoverQuery from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import {useLocation} from 'sentry/utils/useLocation';
+
+type Props = {
+  groupId: string;
+  orgId: string;
+};
+
+function ReplayCount({orgId, groupId}: Props) {
+  const location = useLocation();
+
+  const eventView = EventView.fromNewQueryWithLocation(
+    {
+      id: '',
+      name: `Errors within replay`,
+      version: 2,
+      fields: ['replayId', 'count()'],
+      query: `issue.id:${groupId} !replayId:""`,
+      projects: [],
+    },
+    location
+  );
+
+  return (
+    <DiscoverQuery eventView={eventView} orgSlug={orgId} location={location} useEvents>
+      {({isLoading, tableData}) => {
+        if (isLoading) {
+          return <Placeholder width="24px" height="14px" />;
+        }
+
+        const replayCount = tableData?.data?.length ?? 0;
+        if (replayCount > 0) {
+          return (
+            <ReplayCountContainer data-test-id="replay-count">
+              <IconPlay size="xs" />
+              {replayCount}
+            </ReplayCountContainer>
+          );
+        }
+
+        return null;
+      }}
+    </DiscoverQuery>
+  );
+}
+
+const ReplayCountContainer = styled('div')`
+  display: flex;
+  color: ${p => p.theme.gray400};
+  font-size: ${p => p.theme.fontSizeSmall};
+  gap: 0 2px;
```

Comment:
The existing comment icon uses `gap: 4px;` probably fine to pick that too. The figma is rendered at 5.0 

| 2px | 4px |
| --- | --- |
| <img width="117" alt="Screen Shot 2022-10-26 at 1 21 07 PM" src="https://user-images.githubusercontent.com/187460/198129155-bb047c08-5e6c-4084-bcaa-549343757359.png"> | <img width="115" alt="Screen Shot 2022-10-26 at 1 21 00 PM" src="https://user-images.githubusercontent.com/187460/198129157-de87a97f-7282-4778-96cb-96bdc8618fd9.png"> |